test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -187,6 +187,59 @@ public void KmeansOnnxConversionTest()
             Done();
         }
 
+        [LightGBMFact]
```

Comment:
If we add LightGBMFact to the whole test, this test won't run on x86 builds at all even for the trainers that are supported on x86. A better approach would be to do runtime checking like IsOnnxRuntimeSupported. 
For an example, see MulticlassTrainersOnnxConversionTest() in my pull request for multiclass classifiers. [https://github.com/dotnet/machinelearning/pull/4462](https://github.com/dotnet/machinelearning/pull/4462)

Reply:
Thanks! Made the change.